docs/config.md

```diff
@@ -410,3 +410,10 @@ export default {
 - **Default:** `false`
 
   disable svelte compile statistics.
+
+### disableSvelteResolveWarnings
+
+- **Type** `boolean`
+- **Default:** `false`
+
+  disable svelte resolve warnings
```

Comment:
I don't know if we should make this option available. If we do, we should discourage its use
```suggestion
  disable svelte resolve warnings. Note: this is highly discouraged and you should instead fix these packages which will break in the future.
```

Reply:
users cannot fix these packages themselves, so we should give them an option to avoid this. 

Maybe we can change it to a list so they would still see it for newly added dependencies?

The option itself is experimental so that we can adapt on feedback

Reply:
it's not very noisy since it only happens once per build. and keeping the pressure on package authors seems desirable